lib/stream_data.ex

```diff
@@ -2076,8 +2076,10 @@ defmodule StreamData do
     :rand.seed_s(exported_seed)
   end
 
-  @compile {:inline,
-            split_seed: 1, order: 2, uniform_in_range: 3, lazy_tree: 2, lazy_tree_constant: 1}
+  @compile {
+             :inline,
```

Comment:
Is this being caused by different Elixir versions?

Reply:
Yes, I'm fixing that.